application/dash_application/views/call_ready_ts.py

```diff
@@ -408,67 +394,25 @@ def layout(query_string):
                     core.Tabs([
                         # Graphs tab
                         core.Tab(label="Graphs",
                         children=[
                             core.Graph(
-                                id=ids["total-reads"],
-                                figure=generate_total_reads(df, util.ml_col,
-                                    special_cols["Total Reads (Passed Filter)"],
-                                    initial["colour_by"], initial["shape_by"], initial["shownames_val"],
-                                    [(cutoff_pf_reads_normal_label, initial[cutoff_pf_reads_normal]),
-                                    (cutoff_pf_reads_tumour_label, initial[cutoff_pf_reads_tumour])])),
-
-                            core.Graph(
-                            id=ids["unique-reads"],
-                            figure=generate_unique_reads(df, initial)),
-
-                            core.Graph(
-                                id=ids["mean-target-coverage"],
-                                figure=generate_mean_target_coverage(df, initial)),
-
-                            core.Graph(
-                                id=ids["callability"],
-                                figure=generate_callability(df, initial)),
-
-                            core.Graph(
-                                id=ids["mean-insert-size"],
-                                figure=generate_mean_insert_size(df, initial)),
-
-                            core.Graph(
-                                id=ids["hs-library-size"],
-                                figure=generate_hs_library_size(df, initial)),
-
-                            # core.Graph(
-                            #     id=ids["duplicate-rate"],
-                            #     figure=generate_duplicate_rate(df, initial)),
-                            #
-                            # core.Graph(
-                            #     id=ids["purity"],
-                            #     figure=generate_purity(df, initial)),
-                            #
-                            # core.Graph(
-                            #     id=ids["fraction-excluded"],
-                            #     figure=generate_fraction_excluded(df, initial)),
-                            #
-                            # core.Graph(
-                            #     id=ids["at-dropout"],
-                            #     figure=generate_at_dropout(df, initial)),
-                            #
-                            # core.Graph(
-                            #     id=ids["gc-dropout"],
-                            #     figure=generate_gc_dropout(df, initial)),
-                        ]),
-                        # Tables tab
-                        core.Tab(label="Tables",
+                                id=ids["graphs"],
+                                figure=generate_graphs(df, initial, graphs)
+                            ),
+                        ])
+                    ]),
```

Comment:
Tabs() is closed before the second Tab() is opened

Reply:
Fixed